### PR TITLE
misc: Squash migrations before first release

### PR DESCRIPTION
**Pull request by rsempe**
The goal of this task is to clean some legacy migrations.

### CODE REVIEW

**Review by lovrocolic — APPROVED** [review]
I hope so you have tested it locally by dropping your DB :D

File: db/migrate/20220525122759_init_schema.rb [hunk 1]
```diff
@@ -0,0 +1,235 @@
+class InitSchema < ActiveRecord::Migration[7.0]
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:1]: Style/FrozenStringLiteralComment: Missing frozen string literal comment.

File: db/migrate/20220525122759_init_schema.rb [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  def change
+    enable_extension 'pgcrypto'
+
+    create_table "add_ons", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:5]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+      t.uuid "organization_id", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:6]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "name", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:7]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "code", null: false
+      t.string "description"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:9]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+      t.bigint "amount_cents", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:10]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "amount_currency", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:11]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "created_at", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:12]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "updated_at", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:13]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 10]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["organization_id", "code"], name: "index_add_ons_on_organization_id_and_code", unique: true
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:14]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 11]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["organization_id"], name: "index_add_ons_on_organization_id"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:15]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 12]
```diff
@@ -0,0 +1,235 @@
...
+    end
+
+    create_table "applied_coupons", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:18]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 13]
```diff
@@ -0,0 +1,235 @@
...
+      t.uuid "coupon_id", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:19]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 14]
```diff
@@ -0,0 +1,235 @@
...
+      t.uuid "customer_id", null: false
+      t.integer "status", default: 0, null: false
+      t.integer "amount_cents", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:22]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 15]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "amount_currency", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:23]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 16]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "created_at", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:24]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 17]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "updated_at", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:25]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 18]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "terminated_at", precision: nil
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:26]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 19]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["coupon_id", "customer_id"], name: "index_applied_coupons_on_coupon_id_and_customer_id", unique: true, where: "(status = 0)"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:27]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:27]: Layout/LineLength: Line is too long. [139/120] (https://rubystyle.guide#max-line-length)

File: db/migrate/20220525122759_init_schema.rb [hunk 20]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["coupon_id"], name: "index_applied_coupons_on_coupon_id"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:28]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 21]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["customer_id"], name: "index_applied_coupons_on_customer_id"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:29]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 22]
```diff
@@ -0,0 +1,235 @@
...
+    end
+
+    create_table "billable_metrics", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:32]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 23]
```diff
@@ -0,0 +1,235 @@
...
+      t.uuid "organization_id", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:33]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 24]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "name", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:34]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 25]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "code", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:35]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 26]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "description"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:36]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 27]
```diff
@@ -0,0 +1,235 @@
...
+      t.jsonb "properties", default: {}
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:37]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 28]
```diff
@@ -0,0 +1,235 @@
...
+      t.integer "aggregation_type", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:38]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 29]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "created_at", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:39]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 30]
```diff
@@ -0,0 +1,235 @@
...
+      t.datetime "updated_at", null: false
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:40]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 31]
```diff
@@ -0,0 +1,235 @@
...
+      t.string "field_name"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:41]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 32]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["organization_id", "code"], name: "index_billable_metrics_on_organization_id_and_code", unique: true
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:42]: Style/WordArray: Use `%w` or `%W` for an array of words. (https://rubystyle.guide#percent-w)

Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:42]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 33]
```diff
@@ -0,0 +1,235 @@
...
+      t.index ["organization_id"], name: "index_billable_metrics_on_organization_id"
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:43]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

File: db/migrate/20220525122759_init_schema.rb [hunk 34]
```diff
@@ -0,0 +1,237 @@
...
+    create_table 'applied_coupons', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'coupon_id', null: false
+      t.uuid 'customer_id', null: false
+      t.integer 'status', default: 0, null: false
+      t.integer 'amount_cents', null: false
+      t.string 'amount_currency', null: false
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.datetime 'terminated_at', precision: nil
+      t.index %w[coupon_id customer_id], name: 'index_applied_coupons_on_coupon_id_and_customer_id', unique: true, where: '(status = 0)'
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:29]: Layout/LineLength: Line is too long. [136/120] (https://rubystyle.guide#max-line-length)

File: db/migrate/20220525122759_init_schema.rb [hunk 35]
```diff
@@ -0,0 +1,237 @@
...
+      t.string 'code'
+      t.integer 'status', default: 0, null: false
+      t.datetime 'terminated_at'
+      t.bigint 'amount_cents', null: false
+      t.string 'amount_currency', null: false
+      t.integer 'expiration', null: false
+      t.integer 'expiration_duration'
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index %w[organization_id code], name: 'index_coupons_on_organization_id_and_code', unique: true, where: '(code IS NOT NULL)'
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:72]: Layout/LineLength: Line is too long. [132/120] (https://rubystyle.guide#max-line-length)

File: db/migrate/20220525122759_init_schema.rb [hunk 36]
```diff
@@ -0,0 +1,237 @@
...
+      t.uuid 'customer_id', null: false
+      t.string 'transaction_id', null: false
+      t.string 'code', null: false
+      t.jsonb 'properties', default: {}, null: false
+      t.datetime 'timestamp', precision: nil
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index ['customer_id'], name: 'index_events_on_customer_id'
+      t.index %w[organization_id code], name: 'index_events_on_organization_id_and_code'
+      t.index %w[organization_id transaction_id], name: 'index_events_on_organization_id_and_transaction_id', unique: true
```
Comment (jdenquin) [inline db/migrate/20220525122759_init_schema.rb:121]: Layout/LineLength: Line is too long. [122/120] (https://rubystyle.guide#max-line-length)

**Review by vincent-pochet — APPROVED** [review]

**Pull request merged**